src/bentoml/_internal/cloud/schemas.py

```diff
@@ -452,7 +452,7 @@ class RequestQueueConfig:
 
 @attr.define
 class TrafficControlConfig:
-    timeout: t.Optional[int] = attr.field(default=None)
+    timeout: t.Optional[str] = attr.field(default=None)
```

Comment:
it should be int, not string. You can then write the converter to convert it to string